src/components/wish-item.jsx

```diff
@@ -4,14 +4,15 @@ import ReactStars from "react-rating-stars-component";
 const characterImages = require.context('../assets/images/characters');
 const weaponImages = require.context('../assets/images/weapons');
 export default function WishItem(props) {
-  const { isNewItem } = props
+  const { isNewItem, characterPercentX } = props
   const {src, name, rating, type} = props.item
   return (
     <Col
     xs="2"
     md="1"
     style={{
-      backgroundImage: `url('${type === 'character' ? characterImages('./' + src).default : weaponImages('./' + src).default}')`
+      backgroundImage: `url('${type === 'character' ? characterImages('./' + src).default : weaponImages('./' + src).default}')`,
+      backgroundPositionX: characterPercentX+'%'
```

Comment:
the backgroundPositionX doesn't effect weapons right?

Reply:
I kept the characters.json and updated the rest of the character data in the banners and the code in beginner's wish to have percentX and tested it again and it still looks good. Weapons are set to 50%, which is what is was before. Nevertheless, characterPercentX does set the horizontal offset for weapons so I renamed it to itemPercentX.